x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/settings/settings.tsx

```diff
@@ -238,6 +221,14 @@ export const SettingsPage: React.FC<Props> = memo(
 
     const isUpdating = installationStatus === InstallStatus.installing && installedVersion;
 
+    if (changelogError) {
+      notifications.toasts.addError(changelogError, {
```

Comment:
Probably worth having this in a useEffect has we only want to call the `notifications.toasts.addError` once when the error happen